pages/common/git-checkout-index.md

```diff
@@ -0,0 +1,20 @@
+# git checkout-index
+
+> Copy files from the index to the working tree.
+> More information: <https://git-scm.com/docs/git-checkout-index>.
+
+- Restore any files deleted since last commit:
+
+`git checkout-index --all`
+
+- Restore any files deleted or changed since last commit:
+
+`git checkout-index --all --force`
+
+- Restore any files changed since last commit (ignoring deleted files):
+
+`git checkout-index --all --force --no-create`
+
+- Export a copy of the entire tree at the last commit in the specified directory (the trailing slash is important):
+
+`git checkout-index --all --force --prefix={{git-index-export/}}`
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
`git checkout-index --all --force --prefix={{path/to/export_directory/}}`
```